### ISSUE TITLE
Character Design Tool

### ISSUE THREAD

**Issue by donat-m** [donat-m/Zuerich3D#8]
- Create a modular configurable Character System
- Include ethnic types of Europe
- Your Job ist to program in C# the deformation of the skeleton or skin basic animation produced by Mixiamo or other tools to do rigs. 
- Basic Characters provided by @atepkinniku 
- Estimate: Ok about the 30+ hours for the character design tool
- **TODO**: _Split issue in smaller tasks_

**Comment by donat-m** [comment]
![Character Design Tool](https://user-images.githubusercontent.com/52152977/70130121-4788e400-1680-11ea-9d32-3c00de301e88.png)

Parameters:
- Sex
- Age
- Ethnic type
- Eyes
- Nose
- Mouth
- Hairs
- Ears
- Posture
- Size

Links:
- https://helpx.adobe.com/be_en/adobe-character-animator/how-to/adobe-character-animator.html
- https://tutsplus.com/tutorials/search/character+design
- https://flash-powertools.com/character-rigging-for-flash-animation/
- https://tips.clip-studio.com/en-us/articles/1870

**Comment by donat-m** [comment]
I have some questions. how many characters and how many expression for 3D characters? is it for Unity too?

**Comment by donat-m** [comment]
### How Many Characters ?
- The two man and woman

### How Many Expressions ?
- What do you mean by how many expressions ?

### Its for Unity3D ?
- yes

**Comment by atepkinniku** [comment]
oh sorry, I thought its involved of expressions. I mean like angry, sad, happy, goofy etc

**Comment by atepkinniku** [comment]
completed

**Comment by atepkinniku** [comment]
![characters](https://user-images.githubusercontent.com/50263979/70768576-62520b80-1d98-11ea-8743-16742c070be1.jpg)

Hello, I send you the characters. let me know what do you think? thanks

**Comment by donat-m** [comment]
Hello Atem can you also provide this style of characters https://tips.clip-studio.com/en-us/articles/1801

**Comment by atepkinniku** [comment]
Yes I can,  but it will take some hours as not provided on Adobe Fuse and Mixamo.  I can create 4 characters but the thing is Upwork temporarily suspended on your project so maybe you should fix this problem?

**Comment by donat-m** [comment]
![Screenshot 2019-12-15 20 45 23](https://user-images.githubusercontent.com/52152977/70867997-1790e900-1f7c-11ea-9690-08347a8546fc.png)

**Comment by atepkinniku** [comment]
I have some issues first, I cant work as Upwork is temporarily suspended until now.
second, the sample that you show me is 2D Japanese Anime characters so I need to create 3D characters from zero as Mixamo doesnt provide japanese Anime style and It will take 12+ hours for 2 characters.

I saw also Balsamiq mockups of the client app on the attachment. So How many characters that I have to create?
![Zürich3D](https://user-images.githubusercontent.com/50263979/71145689-fe768980-2254-11ea-88b5-67299ecf2a5f.png)
